src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        DifferentColors[] differentColors = DifferentColors.values();
+        Random random = new Random();
```

Comment:
Consider moving the creation of the 'Random' instance to a class level field, this way you won't create a new instance every time the 'getRandomColor' method is called.